bitnami/airflow/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 annotations:
   category: WorkFlow
 apiVersion: v2
-appVersion: 2.0.2
+appVersion: 2.0.3
```

Comment:
You shouldn't bump the `appVersion` but `version` instaed.

Reply:
Ok, done